test/evergreen.yml

```diff
@@ -2232,7 +2229,6 @@ tasks:
       - func: "compile wiredtiger"
       - func: "make check directory"
         vars:
-          # Run the first test, and then every second test
           extra_args: -L long_running
```

Comment:
Question: Is there a reason we don't run these csuite tests in parallel? It'd bring runtime down from ~1 hour to ~10 minutes

Reply:
I don't think there is any reason not to. Could we give it a go in a patch build?

Reply:
There's nothing in the original PR to suggest issues and my patch passes with a 40 minute time save. I've added the parallel flag in [7a6fbaa](https://github.com/wiredtiger/wiredtiger/pull/9938/commits/7a6fbaacc5519c86f2d5a3c79b7b30687a7308a6)